scripts/buildApiDocs/buildReference.ts

```diff
@@ -93,7 +93,7 @@ export async function buildReference() {
     const json: ComponentDef = {
       name: componentData.name,
       description: descriptionData.componentDescription,
-      props,
+      props: Object.fromEntries(Object.entries(props).sort()),
```

Comment:
If an "overrides" is added manually like `children` (and isn't actually overriding anything generated) the props would not be sorted alphabetically